pkgs/top-level/all-packages.nix

```diff
@@ -5988,8 +5988,6 @@ in
   spidermonkey_31 = callPackage ../development/interpreters/spidermonkey/31.5.nix { };
 
   supercollider = callPackage ../development/interpreters/supercollider {
-    gcc = gcc48; # doesn't build with gcc49
-    qt = qt4;
     fftw = fftwSinglePrec;
```

Comment:
This line seems not needed (anymore?). 
supercollider is building again with fftw single precision, after I took it out.